src/gui/sdl_mapper.cpp

```diff
@@ -3314,8 +3355,6 @@ void MAPPER_BindKeys(Section* sec)
 
 	if (SDL_GetModState()&KMOD_NUM)
 		MAPPER_TriggerEvent(num_lock_event, false);
-
-	GFX_RegenerateWindow(sec);
```

Comment:
Was this the line that was re-initing the video when a joystick got hot-plugged?  I tested on Linux and it's not doing that for me so good job if so :+1:

Reply:
Yeah. I didn't want to change too much, so this still gets called after this function for non-hotplug cases.